lib/theme_settings_validator.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+# Service class that holds helper methods that can be used to validate theme settings.
+class ThemeSettingsValidator
```

Comment:
Not adding test to this service class for now as we already have tests coverage in the spots which is now calling this class.